File: ios/Engine/Providers/AuthProvider.swift
//
//  AuthProvider.swift
//  Glimpse
//
//  Created by Nitin Issac Joy on 10/07/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

import Foundation

class RegisterUserModal {
  var Phone: String = ""
  var Email: String = ""
  var Password: String = ""
}

class LoginUserModal {
  var Email: String = ""
  var Password: String = ""
}

class AuthProvider {
  
  static var Token: String = "";
  
  func setAuthToken(token: String) {
    AuthProvider.Token = token
  }
}

File: android/app/src/main/java/com/beerwithai/glimpse/Engine/Providers/AuthProvider.java
package com.beerwithai.glimpse.Engine.Providers;


class RegisterUserModal {
    String Phone = "";
    String Email = "";
    String Password =  "";
}

class LoginUserModal {
    String Email = "";
    String Password = "";
}

public class AuthProvider {

    public static String AuthToken = "";

    public static void SetAuthToken(String token) {

    }
}
